### ISSUE TITLE
Being able to see the strip/print it after it's already been viewed

### ISSUE THREAD

**Issue by danlovett** [GoodCrossing/StripMaker#7]
Hello!

### Regards to the TAG Item
In the departure list, once 'Print' has been hit and the strip has been closed, 'Printed' will be displayed with the user no longer being able to access it.

### The Suggestion
Maybe the strip should still be able to be viewed after this?
At the moment, the plugin must be reloaded to review the data.

### Recording
![Issue#7](https://user-images.githubusercontent.com/51034848/103468570-4b3edc80-4d52-11eb-8a3c-055c64063571.gif)

:)

**Comment by GoodCrossing** [comment]
Hi! This wasn't really mean to work this way, sorry! Basically, my intentions were to disable this feature for this version that can't print, but I enabled it as part of a bug fix and didn't remove it. The reason this feature was implemented was to avoid misclicking on a strip and printing it again, to reduce paper waste.

I think there could be a simple solution: I can add another TAG function that opens an 'advanced options' drop-down that lets you 'force reprint' if needed. It can also come in handy for other options later on, say, force printing a departure flightplan as an arrival/enroute strip, for example. Sounds good?
